### PR TITLE
Update requirements to support CMS5

### PR DESCRIPTION
**Pull request by monkeyfeet**
Upgraded requirements to support CMS5.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,14 +22,14 @@
     "email": "info@signify.co.nz"
   },
   "require": {
-    "silverstripe/cms": "^4",
-    "dnadesign/silverstripe-elemental": "^4",
-    "symbiote/silverstripe-gridfieldextensions": "^3",
+    "silverstripe/cms": "^5",
```
Comment (sig-steve) [inline composer.json]: Could you please change the constraints to `^4 || ^5`; there's no need to shut out Silverstripe 4 sites if there are no problems with compatibility.

File: composer.json [hunk 2]
```diff
@@ -22,14 +22,14 @@
...
+    "dnadesign/silverstripe-elemental": "^5",
+    "symbiote/silverstripe-gridfieldextensions": "^4",
```
Comment (sig-steve) [inline composer.json]: And please change this to `^3 || ^4`.

File: composer.json (hunk 1 above)
Comment (monkeyfeet) [reply composer.json]: Thanks @sig-steve , good call.

**Review by sig-steve — APPROVED** [review]
I'm happy with these changes.

**Pull request merged**